docs/ios/deployment/overview.md

```diff
@@ -0,0 +1,193 @@
+
+## Create a distribution profile
+
+To publish a .NET MAUI iOS app, you'll need to build a *Distribution Provisioning Profile* specific to it. This profile enables the app to be digitally signed for release so that it can be installed on an iOS device. A distribution provisioning profile contains an app ID and a distribution certificate. For more information, see [Create a distribution profile](provisioning.md#create-a-distribution-profile).
+
+## Download the provisioning profile on your Mac build host
+
+After creating the provisioning profile, it must be added to your Mac build host. The profile can be downloaded to your Mac build host with the following steps:
+
+1. On your Mac, launch Xcode.
+1. In Xcode, select the **Xcode > Preferences...** menu item.
+1. In the **Preferences** dialog, select the **Accounts** tab.
+1. In the **Accounts** tab, click the **+** button to add your Apple Developer Account to Xcode:
+
+    :::image type="content" source="media/overview/accounts-dialog.png" alt-text="Xcode Accounts dialog in preferences.":::
+
+1. In the account type popup, select **Apple ID** and then click the **Continue** button:
+
+    :::image type="content" source="media/overview/account-type.png" alt-text="Xcode select the type of account you'd like to add popup.":::
+
+1. In the sign in popup, enter your Apple ID and click the **Next** button.
+1. In the sign in popup, enter your Apple ID password and click the **Next** button:
+
+    :::image type="content" source="media/overview/sign-in.png" alt-text="Xcode Apple account sign-in.":::
+
+1. In the **Accounts** tab, click the **Manage Certificates...** button to ensure that your distribution certificate has been downloaded.
+1. In the **Accounts** tab, click the **Download Manual Profiles** button to download your provisioning profiles:
+
+    :::image type="content" source="media/overview/account-details.png" alt-text="Xcode Apple Developer Program account details.":::
+
+1. Close Xcode.
+
+## Add entitlements to your app
+
+In iOS, apps run in a sandbox that provides a set of rules that limit access between the app and system resources or user data. *Entitlements* are used to request the expansion of the sandbox to give your app additional capabilities. Any entitlements used by your app must be specified in an entitlements file. For more information about entitlements, see [Entitlements](entitlements.md).
+
+## Add code signing data to your app project
+
+There are project-level settings you must set to sign your iOS app with its provisioning profile. These settings are configured in a `<PropertyGroup>` node:
+
+- `<RuntimeIdentifier>` – the runtime identifier (RID) for the project. Set to `ios-arm64`.
+- `<CodesignKey>` – the name of the distribution certificate you installed into Keychain Access on your Mac build host.
+- `<CodesignProvision>` – the provisioning profile name.
```

Comment:
"This is the name you entered in the Apple Developer portal while creating the provisioning profile"?